src/garage/tf/policies/categorical_cnn_policy.py

```diff
@@ -152,6 +153,8 @@ def get_action(self, observation):
             dict(numpy.ndarray): Distribution parameters.
 
         """
+        observation = flatten_if_unflattened(self.observation_space,
```

Comment:
This is really confusing. Basically, the code here looks like it flattens images, but the code for doing it won't work correctly (because unflattened images are still `np.ndarray`s), and then the next line requires unflattened images. I think the right thing to do here is just to not flatten the observations.